content/administration/upgrade/odoo_online.rst

```diff
@@ -11,15 +11,18 @@ Requesting a test upgrade
    .. image:: odoo_online/accessing-my-databases.png
       :alt: Selecting My Databases under my profile
 
-#. Click on the action settings icon next to your main database and select the *Upgrade* option.
+#. Click on the :guilabel:`Up arrow button` next to your main database name to proceed to the test
+   upgrade.
 
    .. image:: odoo_online/upgrade-option.png
```

Comment:
the new image seems blurry.
Also, we can replace the name of the database with the chrome's inspection tool, instead of blurring the name.
e.g., `example-database`

Reply:
I received the screenshot via discord from the upgrade team as I don't have access to databases. I tried to re do the screenhsot of the screenshot opening the image on my other screen. Let me know if less blurry now. From my side it looks ok.